server/locks_controller.go

```diff
@@ -16,6 +16,7 @@ import (
 // LocksController handles all requests relating to Atlantis locks.
 type LocksController struct {
 	AtlantisVersion    string
+	AtlantisURL        *url.URL
```

Comment:
Passing around the reference because that's what's returned from `url.Parse` and would rather trust that they return a reference for a reason.